component/src/test/java/org/ballerinalang/kafka/nativeimpl/consumer/KafkaConsumerUnsubscribeTest.java

```diff
@@ -0,0 +1,95 @@
+/*
+ *   Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.kafka.nativeimpl.consumer;
+
+import io.debezium.kafka.KafkaCluster;
+import io.debezium.util.Testing;
+import org.ballerinalang.connector.api.BLangConnectorSPIUtil;
+import org.ballerinalang.kafka.util.KafkaConstants;
+import org.ballerinalang.launcher.util.BCompileUtil;
+import org.ballerinalang.launcher.util.BRunUtil;
+import org.ballerinalang.launcher.util.CompileResult;
+import org.ballerinalang.model.values.BBoolean;
+import org.ballerinalang.model.values.BMap;
+import org.ballerinalang.model.values.BValue;
+import org.ballerinalang.util.codegen.ProgramFile;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Properties;
+
+import static org.ballerinalang.kafka.util.KafkaConstants.KAFKA_NATIVE_PACKAGE;
+
+public class KafkaConsumerUnsubscribeTest {
+    private CompileResult result;
+    private static File dataDir;
+    private static KafkaCluster kafkaCluster;
+
+    @BeforeClass
+    public void setup() throws IOException {
+        Properties prop = new Properties();
+        kafkaCluster = kafkaCluster().deleteDataPriorToStartup(true)
+                .deleteDataUponShutdown(true).withKafkaConfiguration(prop).addBrokers(1).startup();
+    }
+
+    @Test(
+            description = "Test functionality of getAvailableTopics() function",
+            sequential = true
```

Comment:
We may need to use singleThreaded instead of sequential because sequential is deprecated.
http://testng.org/doc/documentation-main.html#annotations

Reply:
Noted. Will do the changes accordingly.